.nvmrc

```
@@ -0,0 +1 @@
+18.17.0
```

after #1087 this should be
```suggestion
18.17.1
```

---

updated in 9e5b850